client/app/models/step.coffee

```diff
@@ -0,0 +1,17 @@
+Backbone = require 'backbone'
+
+# StepModel
+# Backbone wrapper for Onboarding's Step models
+# This Class is a proxy between Backone's logic and native Onboarding Steps
+module.exports = class StepModel extends Backbone.Model
+
+    # Map needed property to current model
+    initialize: ({step}) ->
```

Comment:
I though the `Step` class is an internal class for `Onboarding`, and isn't exposed outside of it? Is it not the case now?